tests/perf-system/submitter/handle_arguments.h

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#pragma once
+
+#include "ccf/ds/logger.h"
+
+#include <CLI11/CLI11.hpp>
+#include <iostream>
+
+class ArgumentParser
+{
+public:
+  std::string label; //< Default set in constructor
+  std::string pid_file; //< Default set in constructor
+
+  std::string cert;
+  std::string key;
+  std::string rootCa;
+  std::string server_address = "127.0.0.1:8000";
+  std::string send_filepath = "../tests/perf-system/submitter/cpp_send.parquet";
+  std::string response_filepath =
+    "../tests/perf-system/submitter/cpp_respond.parquet";
+  std::string generator_filepath =
+    "../tests/perf-system/generator/requests.parquet";
+  bool pipeline = false;

```

Comment:
This would be better as max_inflight_requests, where =0 is pipeline=false, an amount can be specified, and -1 means no backpressure.

Reply:
I took a decision at this point about `max_inflight_requests !=0`. Instead of using one loop and overcomplicating the if statement of when to read based on the `max_inflight_requests <0` or `max_inflight_requests>0, I am making this comparison before the loop to avoid overcomplicating once the submission has started.